### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by NeverovDS**
Function that returns the longest word from
the slice of strings.

### CODE REVIEW

**Comment by burov** [comment]
Please use reasonable title

File: longestString.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package main
+
+import "fmt"
+
+func max(slice []string) string {
+	longestWord := 0
```
Comment (burov) [inline longestString.go]: Name does not represent meaning of the variable

File: longestString.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	b := ""
```
Comment (burov) [inline longestString.go]: if variable used across whole function I'd recommend to see reasonable name

File: longestString.go [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	for i := 0; i < len(slice); i++ {
+
+		if longestWord < len(slice[i]) {
```
Comment (imatsiash-ib) [inline longestString.go]: Please use `utf8.RuneCountInString()` function to get the length of utf-8 string.

Comment (NeverovDS) [reply longestString.go]: complete

**Pull request closed without merging**